daemon/libnetwork/internal/nftables/nftables_linux.go

```diff
@@ -193,6 +191,7 @@ type table struct {
 	MustFlush      bool
 
 	applyLock sync.Mutex
+	nftHandle any // applyLock must be held to access
```

Comment:
I have an idea on how to make this field concretely typed.
```suggestion
	nftHandle nftHandle // applyLock must be held to access
```

```go
// nft_cgo_linux.go
type nftHandle = *C.struct_nft_ctx
```
```go
// nft_exec_linux.go
type nftHandle = struct{}
```